examples/tokenserver/src/main/kotlin/pro/streem/examples/tokenserver/Application.kt

```diff
@@ -0,0 +1,36 @@
+package pro.streem.examples.tokenserver
+
+import io.github.cdimascio.dotenv.dotenv
+import io.ktor.application.install
+import io.ktor.auth.Authentication
+import io.ktor.auth.UserIdPrincipal
+import io.ktor.auth.basic
+import io.ktor.features.CallLogging
+import io.ktor.features.ContentNegotiation
+import io.ktor.serialization.json
+import io.ktor.server.engine.embeddedServer
+import io.ktor.server.netty.Netty
+
+fun main() {
+    val dotenv = dotenv()
+
+    embeddedServer(Netty, port = 8080, host = "0.0.0.0") {
```

Comment:
We could pick the port & host values from the .env and default to 8080 & 0.0.0.0 if not provided.

Reply:
Yeah I was going to add that but this PR was already taking longer than I had planned. So I decided to wrap up with what I have so far. We can add this in another PR later.